source/usr_section/usr_manual/gui.rst

```diff
@@ -2,19 +2,21 @@
 
 .. _gui:
 
+=======
 The GUI
-*******
-
-.. figure:: ../../img/manual_gui.png
-   :width: 100%
-
+=======
 
+    .. figure:: /img/manual_gui.png
+       :align: center
 
 Toolbar
 =======
 
 In the toolbar you can find the most common tasks. See table below for information on different buttons and their functionality.
```

Comment:
maybe also describe the functionality that by right-clicking on the toolbar you can add/remove sections